nilearn/_utils/fixes/__init__.py

```diff
@@ -1,24 +1,46 @@
 from distutils.version import LooseVersion
 import sklearn
 
+
 if (LooseVersion(sklearn.__version__) < LooseVersion('0.15') or
       sklearn.__version__ == '0.15-git'):
     from .sklearn_f_regression_nosparse import (
         f_regression_nosparse as f_regression)
 else:
     from sklearn.feature_selection import f_regression
 
+
+try:
+    # scikit-learn < 0.16
+    from sklearn.cross_validation import _check_cv as check_cv
+except:
+    # scikit-learn >= 0.16
+    from sklearn.cross_validation import check_cv
+
+
 # atleast2d_or_csr
 try:
     from sklearn.utils import atleast2d_or_csr
 except ImportError:
     # Changed in 0.15
     from sklearn.utils import check_array as atleast2d_or_csr
 
+
 # roc_auc_score
 try:
     from sklearn.metrics import roc_auc_score
 except ImportError:
     from sklearn.metrics import auc as roc_auc_score
 
-__all__ = ['f_regression', 'atleast2d_or_csr', 'roc_auc_score']
+
+try:
+    from sklearn.utils import check_X_y
+    from sklearn.utils import check_is_fitted
+except ImportError:
+    # scikit-learn < 0.16
+    from .scikit_learn_validation import check_X_y
```

Comment:
May be we should rename ```scikit_learn_validation``` to more appropriate naming depending upon usage of ```check_X_y``` and ```check_is_fitted```. So that it can be used for other purposes wherever necessary. Any comments on this ?